components/resources/ai_chat_ui_strings.grdp

```diff
@@ -420,6 +423,9 @@
   <message name="IDS_CHAT_UI_CHAT_QWEN_SUBTITLE" desc="A description for qwen model" formatter_data="webui=AiChat">
       General purpose chat, coding, and more
   </message>
+  <message name="IDS_CHAT_UI_CHAT_GEMMA_SUBTITLE" desc="A description for gemma model" formatter_data="webui=AiChat">
+    General purpose, text and image understanding
```

Comment:
@Steeeephen could you please add a screenshot to the PR showing the strings in the UI. We need those for localizers. Thanks!

Reply:
<img width="317" alt="image" src="https://github.com/user-attachments/assets/4626fb3e-535b-40ae-a39c-4e5040acd09f" />
<img width="317" alt="image" src="https://github.com/user-attachments/assets/249b9a4f-4b10-4b01-95ae-a3576fad65a3" />